### PR TITLE
Yeelight Matter Request

### PR DESCRIPTION
**Pull request by jacobvoegele**
Pull request for Yeelight Cubes.

### CODE REVIEW

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 1]
```diff
@@ -33,6 +33,13 @@ matterManufacturer:
     productId: 0xAC
     deviceProfileName: plug-binary
 
+#Yeelight
+  - id: "Yeelight Smart Lamp"
+    deviceLabel: Yeelight Smart Lamp
+    vendorId: 0x1312
+    productId: 0x1
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/fingerprints.yml:40]: nit: I would add a leading zero to make this `0x01` to for consistency with the other fingerprints above

File: drivers/SmartThings/matter-switch/profiles/light-level-colorTemperature-2700k-6500k.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+name: light-level-colorTemperature-2700k-6500k
```
Comment (ctowns) [inline drivers/SmartThings/matter-switch/profiles/light-level-colorTemperature-2700k-6500k.yml:1]: This profile is a duplicate of the profile `light-color-level-2700K-6500K` that was introduced here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/872/files

Please delete this file and then update your repo to include the latest changes, then point your fingerprint to the duplicate profile that already exists. Then this should be good to go!

File: drivers/SmartThings/matter-switch/fingerprints.yml (hunk 1 above)
Comment (jacobvoegele) [reply drivers/SmartThings/matter-switch/fingerprints.yml]: Fixed

**Review by ctowns — APPROVED** [review]
Looks great, thanks Jacob!

**Pull request merged**